src/main/java/com/ibm/cloudant/kafka/connect/CloudantSourceConnectorConfig.java

```diff
@@ -29,64 +27,72 @@
 
-				// Kafka topic
-				.define(InterfaceConst.TOPIC, Type.LIST,
-						Importance.HIGH,
-						ResourceBundleUtil.get(MessageKey.KAFKA_TOPIC_LIST_DOC),
-						DATABASE_GROUP, 1, Width.LONG,
-						ResourceBundleUtil.get(MessageKey.KAFKA_TOPIC_LIST_DISP))
+                // Cloudant URL
+                .define(InterfaceConst.URL, Type.STRING, Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_URL_DOC),
+                        DATABASE_GROUP, 1, Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_URL_DISP))
+                // Cloudant Username
+                .define(InterfaceConst.USER_NAME, Type.STRING, Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_USR_DOC),
+                        DATABASE_GROUP, 1, Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_USR_DISP))
+                // Cloudant Password
+                .define(InterfaceConst.PASSWORD, Type.PASSWORD, Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_PWD_DOC),
+                        DATABASE_GROUP, 1, Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_PWD_DISP))
+                // Cloudant last change sequence
+                .define(InterfaceConst.LAST_CHANGE_SEQ, Type.STRING, CLOUDANT_LAST_SEQ_NUM_DEFAULT,
+                        Importance.LOW,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_LAST_SEQ_NUM_DOC),
+                        DATABASE_GROUP, 1, Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_LAST_SEQ_NUM_DOC))
 
-				// Whether to generate a struct Schema
-				.define(InterfaceConst.USE_VALUE_SCHEMA_STRUCT, Type.BOOLEAN,
-						false,
-						Importance.HIGH,
-						ResourceBundleUtil.get(MessageKey.CLOUDANT_STRUCT_SCHEMA_DOC),
-						DATABASE_GROUP, 1, Width.NONE,
-						ResourceBundleUtil.get(MessageKey.CLOUDANT_STRUCT_SCHEMA_DISP));
-	}
-	
-	public CloudantSourceConnectorConfig(Map<String, String> originals) {
-		super(CONFIG_DEF, originals, false);
-	}
-	
-	protected CloudantSourceConnectorConfig(ConfigDef subclassConfigDef, Map<String, String> originals) {
-		    super(subclassConfigDef, originals);
-	}
-	
-	public static void main(String[] args) {
-		  System.out.println(CONFIG_DEF.toRst());
-	}
+                // Kafka topic
+                .define(InterfaceConst.TOPIC, Type.LIST,
+                        Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.KAFKA_TOPIC_LIST_DOC),
+                        DATABASE_GROUP, 1, Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.KAFKA_TOPIC_LIST_DISP))
+
+                // Whether to generate a struct Schema
+                .define(InterfaceConst.USE_VALUE_SCHEMA_STRUCT, Type.BOOLEAN,
+                        false,
+                        Importance.HIGH,
```

Comment:
Does this need to be `Importance.HIGH` if it's optional?

Reply:
Fixed in e44c82e62ea989e9a697167a8f14e44512609c6c